third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/services/container/resource_container_cluster.go

```diff
@@ -2886,44 +2958,6 @@ func resourceContainerClusterUpdate(d *schema.ResourceData, meta interface{}) er
 		log.Printf("[INFO] GKE cluster %s's binary authorization has been updated to %v", d.Id(), enabled)
 	}
 
-	if d.HasChange("private_cluster_config.0.enable_private_endpoint") {
```

Comment:
Why are we deleting this?  (Or am I misreading the diff?)

Reply:
It's moved up to line 2856. The logic arpimd how the update is handled is changed.